_posts/2021/2021-01-11-read-ruby-version-in-your-gemfile.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Read the .ruby-version into your Gemfile"
+description: "A lovely little improvement to your life when upgrading Ruby"
+layout: article
+category: ruby
+image:
+  base: '2021/read-ruby-version-in-your-gemfile'
+  alt: 'Falling white dice'
+  credit:  Riho Kroll
+  source: "https://unsplash.com/photos/m4sGYaHYN5o"
+
+---
+
+Previously I recommended [using a loose version ruby version constraint in your Gemfile](/ruby/use-loose-ruby-versioning-in-your-gemfile/). This is still a valuable technique, but here’s another useful variation that's good for most of us, most of the time.
+
+Hat tip to [Emma](https://twitter.com/EmmaBeynon) for the this one-liner.
+
+Ruby version managers (such as [chruby](https://github.com/postmodern/chruby)) ensure an application uses a specific version of Ruby by looking for a [`.ruby-version` file](https://gist.github.com/fnichol/1912050) in the root directory of each app.
+
+The file specifies the required version of Ruby for this application and the manager automatically switches the environment to use the specified version.
+
+[The version can also be specified in your Gemfile](https://bundler.io/gemfile_ruby.html) using bundler, which is more often used in a deployed environment.
```

Comment:
Are you saying 'bundler' is more often used in a deployed env, or this technique is?

Reply:
```suggestion
[The version can also be specified in your Gemfile](https://bundler.io/gemfile_ruby.html) using bundler, which is more often used to define the version of Ruby to use in a deployed environment.
```